src/engraving/libmscore/cmd.cpp

```diff
@@ -912,7 +912,9 @@ Segment* Score::setNoteRest(Segment* segment, track_idx_t track, NoteVal nval, F
             tick += ncr->actualTicks();
 
             if (!articulationIds.empty()) {
-                toChord(ncr)->updateArticulations(articulationIds);
+                if (ncr->isChord()) {
+                    toChord(ncr)->updateArticulations(articulationIds);
```

Comment:
Alternatively: do this a few lines higher directly when the chord is created, so that you can avoid the `ncr->isChord()` check and use the `chord` variable instead.

For example, around line 892:
```c++
if (!articulationIds.empty()) {
    chord->updateArticulations(articulationIds);
}
```

Maybe that solution looks a little bit cleaner.